docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -1,17 +1,116 @@
+
+Since Sourcegraph 5.3, every Cody Enterprise setup that uses [Cody Gateway](/cody/core-concepts/cody-gateway) can opt-in to using StarCoder as an alternative to Claude Instant for autocomplete. To ensure a fast and reliable experience, we are partnering with [Fireworks](https://fireworks.ai/) and have set up a dedicated dedicated hardware deployment for our Enterprise users.
+
+To enable StarCoder go to **Site admin > Site configuration** (`/site-admin/configuration`) and change the `completionModel`:
+
+```json
+{
+  // [...]
+  "cody.enabled": true,
+  "completions": {
+    "provider": "sourcegraph",
+    "completionModel": "fireworks/starcoder"
+  }
+}
+```
+
+Cody Extension will automatically pick up this change when they connect to your Enterprise instance.
+
+It's recommended that every instance admin not using a third-party LLM provider makes this change and we are planning to make this the default in a future release.
+
+### Multi-repo context
+
+<Callout type= "info">Multi-repo context for Sourcegraph Enterprise is supported with VS Code and JetBrains IDEs extensions.</Callout>
+
+Cody Enterprise allows up to 10 repos to support multi-repo context for users on IDE extensions. These can be added from the enhanced context selector window. Multi-repo context can search and retrieve context from multiple repositories responding to user questions, opening up more complex use cases. Enterprise users can ask questions requiring broader knowledge from across a codebase.
+
+### Enhanced context selector
+
+<Callout>Enhanced context selector is currently supported on VS Code and JetBrains IDE extensions.</Callout>
+
+The **Enhanced context selector** panel allows Cody Enterprise users to add up to **10 repos** to search across multiple repositories. This will enable users to ask questions requiring broader knowledge from across a codebase.
+
+Create a new chat in VS Code and click the **Enhanced Context** icon in the chat type field. Click the **Choose Repositories...** button and select up to 10 repositories from the list.
+
+![enhanced-context-enterprise](https://storage.googleapis.com/sourcegraph-assets/Docs/enhanced-context-enterprise.png)
+
+### Cody ignore
+
+<Callout type="info">Cody Ignore is currently in the Experimental stage and is only supported in VS Code extension.</Callout>
+
+Users can choose files and folders to exclude as context for Cody. It works as follows:
+
+- `.cody/ignore` allows users to use the familiar `gitignore` file pattern to define files that they do not want Cody to use as context
+- The purpose of `.cody/ignore` is to ensure that certain files are not shared with third parties LLM provider
+- Executing a command on an ignored file will trigger a system pop-up to notify you that Cody is ignoring the file
```

Comment:
```suggestion
```
Cody ignore is still behind the unstable flag in VS Code it feels too soon to document how it works, because following these instructions will not activate it.